app/code/Magento/Developer/Console/Command/DiInfoCommand.php

```diff
@@ -173,4 +195,41 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return \Magento\Framework\Console\Cli::RETURN_SUCCESS;
     }
+
+    /**
+     * Set Area for DI Configuration
+     *
+     * @param string $area
+     * @return void
+     * @throws \InvalidArgumentException
+     */
+    private function setDiArea($area): void
```

Comment:
The parameter $area should have a type hint of string.

Reply:
fixed